README.md

```diff
@@ -51,8 +52,17 @@ Pieman is a script for creating custom OS images for single-board computers such
 
 #### Optional
 
-* bzip2
-* xz
+* To enable `COMPRESS_WITH_BZIP2` and `COMPRESS_WITH_XZ`:
+  * bzip2
+  * xz
+* To enable [Mender](https://mender.io) support:
+  * Development libraries and header files related to C standard library (make sure the package, which is going to be installed to satisfy the dependency, includes `/usr/include/sys/types.h`)
+  * C programming language compiler
+  * Go programming language compiler
+  * bc
+  * dtc
+  * make
+  * mkimage
```

Comment:
Here is the list of packages for Ubuntu 18.04 if you want to add them
`bc gcc device-tree-compiler golang-go make u-boot-tools libc6-dev-i386`